src/network.c

```diff
@@ -105,6 +105,13 @@ static gboolean transfer(RaucTransfer *xfer, GError **error)
 	curl_easy_setopt(curl, CURLOPT_MAXFILESIZE_LARGE, xfer->limit);
 	/* decode all supported Accept-Encoding headers */
 	curl_easy_setopt(curl, CURLOPT_ACCEPT_ENCODING, "");
+	
```

Comment:
Uncrustify reveals that this adds a trailing white space.

Reply:
Okay, removed trailing spaces.